docs/app/src/versions.js

```diff
@@ -12,10 +12,16 @@ angular.module('versions', ['currentVersionData', 'allVersionsData'])
             /** @this VersionPickerController */
             function VersionPickerController($location, $window, CURRENT_NG_VERSION, ALL_NG_VERSIONS) {
 
-      var versionStr = CURRENT_NG_VERSION.isSnapshot ? 'snapshot' : CURRENT_NG_VERSION.version;
+      var versionStr = CURRENT_NG_VERSION.version;
+
+      if (CURRENT_NG_VERSION.isSnapshot) {
+        versionStr = CURRENT_NG_VERSION.distTag === 'latest' ? 'snapshot-stable' : 'snapshot';
```

Comment:
perhaps we could rename `snapshot` and `snapshot-stable` to `next` and `stable` respectively?

Reply:
I don't know. I think it's important that these are snapshots of a branch. "Stable" says "release" to me, not snapshot. "Next" is better, but it also doesn't say it's a snapshot. What about "snapshot-next"?

Reply:
btw, these are not the labels that are shown - for "snapshot" we show "master", and for "snapshot-stable" we show "v1.x.y+1-snapshot"

Reply:
I see. Cool to leave them as-is then.

Reply:
or `master-snapshot` and `stable-snapshot`

Reply:
The labels in the version picker are now "master-snapshot" and e.g. "v1.6.7-snapshot"